test/integration/OracleReader.spec.ts

```diff
@@ -45,19 +50,9 @@ describe('oracledb', () => {
 
             await pool.open();
 
-            // TODO: simplify with .should.be.rejected
-            let rejected = false;
-            const readPromise = reader.read('SELECT * FROM tableDNE');
-            try {
-                await readPromise;
-            } catch (err) {
-                console.log(err);
-                rejected = true;
-            } finally {
-                await reader.close();
-                await pool.close();
-                expect(rejected).to.be.true;
-            }
+            const readPromise = reader.read({ name: 'tableONE', query: 'SELECT * FROM tableDNE' });
+
+            expect(readPromise).to.eventually.be.rejectedWith(Error);
```

Comment:
So excited to see this. For some reason, it was not working for me in VS Code

Reply:
Are we leaking an oracle connection pool during this test, though? That might eventually cause problems for the Oracle server the tests run against

Reply:
Good catch, and it should be fixed now